### PR TITLE
[I10378] Fix alignment in grid panels

### PR DESCRIPTION
**Pull request by ShibumiGrant**
Shibumi Issue: [Form Vertical Dividers Misaligned](https://app.shibumi.com/shibumi/7f60d188-303c-43d7-b2dd-13330ef5dc07/workItem-summary?id=5051fe26-c88e-43e1-848d-681f12382b67)

Remake of https://github.com/shibumicorp/react-grid-layout-legacy/pull/1

`mothership`: https://github.com/shibumicorp/shibumi/pull/8544

### CODE REVIEW

File: lib/calculateUtils.js [hunk 1]
```diff
@@ -84,6 +75,22 @@ export function calcGridItemPosition(
...
+  }
+
+  // If resizing, use the exact width and height as returned from resizing callbacks.
+  if (state && state.resizing) {
+    out.width = Math.round(state.resizing.width);
+    out.height = Math.round(state.resizing.height);
+  } else if (right > 0 && bottom > 0) {
+    out.width = right - out.left;
+    out.height = bottom - out.top;
+  }
```
Comment (shb-rpolasek) [inline lib/calculateUtils.js:89]: Are we always going to have right and bottom > 0 at the same time?  
Would we want to make them independent?
e.g.
```suggestion
  } else  {
     if (right > 0) {
         out.width = right - out.left;     
      }
      if( bottom > 0) {
          out.height = bottom - out.top;
      }
  }
```
```

**Review by shb-rpolasek — APPROVED** [review]
One question because I'm not fully clear on the use cases here.
Otherwise looks good.  Approving.

File: lib/calculateUtils.js (hunk 1 above)
Comment (ShibumiGrant) [reply lib/calculateUtils.js:89]: Seems like it covers all cases that matter - we were using this same code in this PR in the old package for this same fix so I just kept it as-is.

Comment (shb-rpolasek) [reply lib/calculateUtils.js:89]: In that case, carry on.

**Pull request merged**